tests/specs/benchmarks/verification.k

```diff
@@ -60,7 +60,8 @@ module VERIFICATION
   // Rule Replacement
   // ########################
 
-    claim <k> EXTCODESIZE ACCT => #extCodeSize(ACCT) ~> #push ... </k>  [trusted]
+    // TODO: is this needed anymore?
```

Comment:
Is this claim needed for something? Its odd to see a `claim` outside of a `-SPEC` module

Reply:
in this case, the `claim` is acting like a rule basically